### DISCUSSION TITLE
unknown error occured

### DISCUSSION THREAD

**Discussion by Shikhaechovme** [GorvGoyl/ChatGPT-Writer#44] [Report Bugs and Issues]
There was an unknown error. Make sure you are logged-in to chat.openai.com If issue persists, please report it here., Message keeps popping, unable to generate mails

**Comment by Encrypted73** [comment]
Same error even though I was logged into chat.openai.com

**Comment by GorvGoyl** [comment]
fixed in version v2.1.4.. please wait for it to go live on chrome extension store https://chrome.google.com/webstore/detail/chatgpt-writer-email-writ/pdnenlnelpdomajfejgapbdpmjkfpjkp/

**Comment by ASalkever** [comment]
I am receiving the same error but I have the latest version, my Chrome is up to date, and I am logged into ChatGPT.  Is it possible that there is a conflict with another extension? AdBlock or Grammarly?

**Comment by Encrypted73** [comment]
It's back to not working again: (
There was an unknown error. Make sure you are logged-in to [chat.openai.com](https://chat.openai.com/)
If issue persists, please [report it here](https://github.com/GorvGoyl/ChatGPT-Writer/discussions/categories/report-bugs-and-issues).

**Reply by MikeSBDJ** [reply]
The error is still there version 2.1.4 ("There was an unknown error. Make sure you are logged-in to [chat.openai.com](https://chat.openai.com/) If issue persists, please [report it here](https://github.com/GorvGoyl/ChatGPT-Writer/discussions/categories/report-bugs-and-issues).")

Note prior there was a message noting something to the effect of "requires Cloudflare security check"

**Comment by ghost** [comment]
same for me too. I'm logged into chatGPT pro

**Comment by GorvGoyl** [comment]
it's working for me... is the issue happening for just chatGPT pro users?

**Comment by ghost** [comment]
I'm a chat GPT pro users... is this why it's not working?
